tests/devices/librerouter-v1/board.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
isn't this file available somewhere already? can find the same one in the /etc/board.json of the librerouters...

Reply:
So we  can't get it from there for the tests then? (just thinking on deduplicating it because of the implications on maintenance...)

Reply:
I am not sure to be understanding your question. You mean from inside a LinbreRouter? Yes that is what I did (I copied the board.json file and the other two files, network and wireless) and that is what I think we should do for other devices too. I don't think there is an online database of this files so we don't have to do it. If we want to have this kind of integration test "with device emulation" then I don't think we have other options, or maybe I am not seeing it?

Reply:
I mean getting that information from where it comes instead of hardcoding it... 
don't worry, it is a lower priority thing. important to keep an eye on deduplication, but certainly not urgent